src/index.js

```diff
@@ -70,7 +70,12 @@ function addPackageData (inputs, path) {
 function computeHash (pepper, paths, seed) {
   const inputs = []
   if (pepper) inputs.push(pepper)
-  if (seed) inputs.push(seed)
+  if (seed) {
```

Comment:
Done (https://github.com/novemberborn/package-hash/commit/25a55aff14cbf2e94d9bc1a2e8a006901b74373d). You'll have to rebase, sorry.